float32[6] sensor_data
